tools/wpt/expectation.json

```diff
@@ -4732,5 +4732,278 @@
+      "NavigationPreloadManager interface: operation getState()",
+      "Cache interface: existence and properties of interface object",
+      "Cache interface object length",
+      "Cache interface: operation match(RequestInfo, optional CacheQueryOptions)",
+      "Cache interface: operation matchAll(optional RequestInfo, optional CacheQueryOptions)",
+      "Cache interface: operation add(RequestInfo)",
+      "Cache interface: operation addAll(sequence<RequestInfo>)",
+      "Cache interface: operation delete(RequestInfo, optional CacheQueryOptions)",
+      "Cache interface: operation keys(optional RequestInfo, optional CacheQueryOptions)",
+      "Cache interface: self.cacheInstance must inherit property \"matchAll(optional RequestInfo, optional CacheQueryOptions)\" with the proper type",
+      "Cache interface: calling matchAll(optional RequestInfo, optional CacheQueryOptions) on self.cacheInstance with too few arguments must throw TypeError",
+      "Cache interface: self.cacheInstance must inherit property \"add(RequestInfo)\" with the proper type",
+      "Cache interface: calling add(RequestInfo) on self.cacheInstance with too few arguments must throw TypeError",
+      "Cache interface: self.cacheInstance must inherit property \"addAll(sequence<RequestInfo>)\" with the proper type",
+      "Cache interface: calling addAll(sequence<RequestInfo>) on self.cacheInstance with too few arguments must throw TypeError",
+      "Cache interface: self.cacheInstance must inherit property \"keys(optional RequestInfo, optional CacheQueryOptions)\" with the proper type",
+      "Cache interface: calling keys(optional RequestInfo, optional CacheQueryOptions) on self.cacheInstance with too few arguments must throw TypeError",
+      "CacheStorage interface: operation match(RequestInfo, optional MultiCacheQueryOptions)",
+      "CacheStorage interface: operation keys()",
+      "CacheStorage interface: caches must inherit property \"match(RequestInfo, optional MultiCacheQueryOptions)\" with the proper type",
+      "CacheStorage interface: calling match(RequestInfo, optional MultiCacheQueryOptions) on caches with too few arguments must throw TypeError",
+      "CacheStorage interface: caches must inherit property \"keys()\" with the proper type",
+      "WorkerGlobalScope interface: attribute caches",
+      "WorkerNavigator interface: attribute serviceWorker"
+    ],
+    "cache-storage": {
+      "cache-match.https.any.html": [
+        "Cache.match supports ignoreMethod",
+        "Cache.match supports ignoreVary",
+        "Cache.match with Request and Response objects with different URLs",
+        "Cache.match with a network error Response",
+        "cors-exposed header should be stored correctly.",
+        "MIME type should be set from content-header correctly.",
+        "Cache.match ignores vary headers on opaque response."
+      ],
+      "cache-delete.https.any.html": [
+        "Cache.delete called with a HEAD request",
```

Comment:
This sounds like it should be passing.

Reply:
`ignoreMethod` option is not supported hence the test fails